source/hp/dof_handler.cc

```diff
@@ -227,7 +227,7 @@ namespace internal
           vertex_fe_association (dof_handler.finite_elements->size(),
                                  std::vector<bool> (dof_handler.tria->n_vertices(), false));
 
-          for (typename DoFHandler<dim,spacedim>::active_cell_iterator
+          for (TriaActiveIterator < dealii::DoFCellAccessor < DoFHandler < dim, spacedim >, false > >
```

Comment:
Ad "What happens if you use typename hp::DoFHandler<dim,spacedim>::active_cell_iterator...":

Error	1	error C2882: 'DoFHandler' : illegal use of namespace identifier in expression	C:\hpfem\agros2d\dealii\source\hp\dof_handler.cc	590	obj_hp.debug